mypy/build.py

```diff
@@ -848,6 +901,11 @@ def find_cache_meta(id: str, path: str, manager: BuildManager) -> Optional[Cache
                     manager.trace('    {}: {} != {}'
                                   .format(key, cached_options.get(key), current_options.get(key)))
         return None
+    if manager.old_plugins_snapshot and manager.plugins_snapshot:
```

Comment:
What about adding a test for this?

Reply:
We can't yet add tests for daemon. I opened https://github.com/python/mypy/issues/5891 for this.